packages/medusa/src/joiner-configs/cart-service.ts

```diff
@@ -4,7 +4,7 @@ import { ModuleJoinerConfig } from "@medusajs/types"
 import { Cart } from "../models"
 
 export default {
-  serviceName: Modules.CART,
+  serviceName: "cartService",
```

Comment:
q: is this meant to be like this?

Reply:
`Modules.CART` was introduced recently by mistake. These joiner configs are from when we attempted to add joiner configs for internal services, so this should apply to `cartService` as far as I can understand